### PR TITLE
GP2 Biography Modal

### PR DESCRIPTION
**Pull request by markcsims**

### CODE REVIEW

File: packages/gp2-components/src/organisms/BiographyModal.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    >
+      {({ isSaving }) => (
+        <LabeledTextArea
+          value={newBiography}
+          onChange={setNewBiography}
+          enabled={!isSaving}
+          required
+          maxLength={2500}
+          title="Background"
+          subtitle="(Required)"
```
Comment (jhonny-m) [inline packages/gp2-components/src/organisms/BiographyModal.tsx:40]: I think we have them as `(required)` in other places

File: packages/gp2-components/src/organisms/__tests__/BiographyModal.test.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    expect(screen.getByRole('dialog')).toContainElement(
+      screen.getByRole('heading', { name: 'Biography' }),
+    );
+  });
+
+  it('renders biography', () => {
+    renderModal();
+    expect(
+      screen.getByRole('textbox', {
+        name: 'Background (Required)',
```
Comment (jhonny-m) [inline packages/gp2-components/src/organisms/__tests__/BiographyModal.test.tsx:37]: will a regex for background only work here?

File: packages/gp2-components/src/organisms/BiographyModal.tsx (hunk 1 above)
Comment (markcsims) [reply packages/gp2-components/src/organisms/BiographyModal.tsx]: Done

File: packages/gp2-components/src/organisms/__tests__/BiographyModal.test.tsx (hunk 2 above)
Comment (markcsims) [reply packages/gp2-components/src/organisms/__tests__/BiographyModal.test.tsx]: Done

**Review by jhonny-m — APPROVED** [review]

**Pull request merged**